docs/concepts/moneyd.md

```diff
@@ -0,0 +1,19 @@
+# moneyd
+[`moneyd`](https://github.com/interledgerjs/moneyd) is a daemon that enables you to connect to an Interledger node, 
+and to send or receive value in the form of ILP packets to or from that node. `moneyd` functions like your 
+home wifi router in the sense that both transfer packets and connect to an upstream network. However, `moneyd` uses 
+[Interledger packets](https://github.com/interledger/rfcs/blob/master/0027-interledger-protocol-4/0027-interledger-protocol-4.md) 
+which represent value and data, rather than just data.
+
+`moneyd` uses uplinks to communicate with other nodes in an ILP network. An uplink allows moneyd to send messages 
+(ILP packets) to a parent node, and to send/receive settlements (real money) with that parent connector. 
+
+When you run `moneyd` using an uplink, it:
+* Starts an Interledger node
```

Comment:
As per the previous discussion it'd be good if @emschwartz could weigh in here, but I agree with @ryangyoung that connector is a more appropriate and descriptive term than 'node'

Reply:
I don't feel that strongly about this naming issue. Connector is more in line with other docs that we currently have, whereas node is more in line with blockchain ecosystem terminology :man_shrugging: